sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -113,6 +118,11 @@ public HttpMessage CreateMessage(RequestContext? context)
         /// </summary>
         public ResponseClassifier ResponseClassifier { get; }
 
+        /// <summary>
+        /// The <see cref="MessageClassifier"/> instance used in this pipeline invocations.
+        /// </summary>
+        private MessageClassifier? _perClientClassifier { get; }
```

Comment:
I'll try to remove this and have it already on ResponseClassifier - that will also simplify CreateMessage methods.

Reply:
Addressed.